### ISSUE TITLE
Calc: Missing Tracking changes menu items to turn on/off, to accept/reject as well as the Manage dialog

### ISSUE THREAD

**Issue by pedropintosilva** [CollaboraOnline/online#4111]
**Describe the bug**
When working with spreadsheets there is no way to turn on/off Tracking changes via UI
Initially reported here: https://forum.collaboraonline.com/t/how-to-hide-revision-history-on-calc/816/2

**To Reproduce**
1. Open a spreadhsheet
2. Use the tracking changes keyboard shortcut to switch that on: `Ctrl + Shift + V` on
3. 
  - (In the notebookbar mode) Go to hamburger menu (top left):  option is missing  
  - (In the classic mode) Go to Menu > Edit: option is missing

**Expected behavior**
The option should be visible in those places, similar to how it works on text documents

**Actual behavior**
option is missing . The only way to turn it off is by pressing `Ctrl + Shift + C`

**Desktop (please complete the following information)**
 - Master

**Comment by thebearon** [comment]
I'm not sure this should be treated as a bug, rather as a marginal enhancement. @kendy, what's your take on this feature?

**Comment by pedropintosilva** [comment]
@thebearon yeah I see , we first need to be sure that it makes sense to have the tracking changes in the spreadsheets (meaning: if it's stable enough etc)

**Comment by cornouws** [comment]
Sorry for vandalizing the description ;)
The mentioned items are all present in COOL Writer. And do make the feature useful.

**Comment by cornouws** [comment]
Feature wise, I see only one difference in what one can do in online compared to desktop - though I didn't check everything.

- Online one can set and use grouping of columns and rows
- on Desktop, that feature is disabled when Change Tracking is turned on.

**Comment by cornouws** [comment]
> Feature wise, I see only one difference in what one can do in online compared to desktop - though I didn't check everything.
> 
>     * Online one can set and use grouping of columns and rows
> 
>     * on Desktop, that feature is disabled when Change Tracking is turned on.

So I didn't see any possible damage from change tracking being available in online - although managing changes is not available at the moment.

**Comment by smehrbrodt** [comment]
Change tracking items added to menu and Notebookbar with https://gerrit.collaboraoffice.com/plugins/gitiles/online/+/b29f2260860ec83cbf2206a769e8f0a8a927f31a%5E%21/
